### ISSUE TITLE
Make plot attributes histograms - direction

### ISSUE THREAD

**Issue by nicolamos** [Moelf/FHist.jl#104]
I need to plot a 1D histogram vertical with Makie. This is usually accomplished by specifying `direction=:y` to a plot call like `hist` or `density`, however this seems to not be propagated for a Hist1D type.

There are currently workarounds or better ways to achieve this?

**Comment by Moelf** [comment]
yeah it should be propagated to `hist()`, if you can supply an example outcome I'd happy to try to see if I can improve the recipe.

I haven't worked on it again since Makie 0.20 release so other than fixing to make it work

**Comment by nicolamos** [comment]
Sure, for example a simple example can be the following.

```julia
xs = randn(100)
hh = FH.Hist1D(xs)
barplot(hh, direction=:x)
```

vs 

```julia
xs = randn(100)
hh = FH.Hist1D(xs)
plot(hh, direction=:x)
```

There is also a problem of consistency of the `direction` argument in Makie but that’s another story...

**Comment by Moelf** [comment]
this works now in v0.11:

```julia
julia> using FHist, CairoMakie

julia> hh = Hist1D(randn(100))
edges: [-2.5, -2.0, -1.5, -1.0, -0.5, 0.0, 0.5, 1.0, 1.5, 2.0, 2.5]
bin counts: [3.0, 8.0, 6.0, 17.0, 19.0, 20.0, 13.0, 7.0, 6.0, 1.0]
total count: 100.0

julia> plot(hh, direction=:x)
```
![image](https://github.com/Moelf/FHist.jl/assets/5306213/6e30baed-765c-4f0a-90c7-7bf6b53fb8e4)

**Comment by Moelf** [comment]
let me know if you can't migrate to v0.11 -- notice the main constructor APIs are re-designed, it is much cleaner now so you probably should migrate.

But in the unlikely case you can't, I can try to backport the Makie recipe fixes to v0.10 .x

**Comment by nicolamos** [comment]
Hi! Thank you for this update. From my side it is fine, I can migrate to the new version.
